### PR TITLE
chore(admin): convert useInjectionZone to TS

### PR DESCRIPTION
**Pull request by gu-stav**
### What does it do?

Convert `useInjectionZone` to TS.

### CODE REVIEW

File: packages/core/admin/admin/src/shared/hooks/useInjectionZone.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { useAdmin } from '../../contexts/admin';
+
+export type InjectionZoneArea = string;
```
Comment (joshuaellis) [inline packages/core/admin/admin/src/shared/hooks/useInjectionZone.ts:3]: would it be better to do `${string}.${string}.${string}` to give better typesafety?

**Review by joshuaellis — APPROVED** [review]
An idea, but shouldn't block the PR :)

File: packages/core/admin/admin/src/shared/hooks/useInjectionZone.ts (hunk 1 above)
Comment (gu-stav) [reply packages/core/admin/admin/src/shared/hooks/useInjectionZone.ts:3]: I think what would really improve this type is to hard-code the fixed list of injection zones we have. I'll have a look later and see if I can extract it. 👍🏼

**Review by joshuaellis — APPROVED** [review]
mmm very nice ✨

**Pull request merged**